src/main/java/Duke.java

```diff
@@ -1,10 +1,66 @@
+/**
+ * Main driver class for Duke.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    protected Storage storage;
+    protected Ui ui;
+    protected TaskList arrayOfTasks;
```

Comment:
It would be better to adhere to the coding standard and make the attributes private here instead!